pkg/sql/parsers/dialect/mysql/mysql_sql.y

```diff
@@ -3444,6 +3444,12 @@ select_expression:
     }
 |   expression as_name_opt
     {
+        switch v:=$1.(type){
```

Comment:
Parser does not distinguish between null types, and repairs them in the plan

Reply:
I have changed it now, please review again